haiku-apps/wolle/wolle-0.1~git.recipe

```diff
@@ -0,0 +1,45 @@
+srcGitRev="be9165f52bee14d3cd1d19600ce6cfa44f217df4"
+SUMMARY="A screensaver displaying  woollen string"
+DESCRIPTION="Wolle is a screensaver which draws a woollen string over 
```

Comment:
Please add a `\` like `woollen string over \` at the end of the line for line breaks. See the [recipe guidelines](https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines)